### ISSUE TITLE
Error parsing v2d files with slashes in strings

### ISSUE THREAD

**Issue by varenius** [USNO-VLBI/main#4]
I have correlated data with zoom windows. The v2d file contains the section
```
ZOOM zoom
{
addZoomFreq = freq@8212.99/bw@8.0/noparent@true
addZoomFreq = freq@8252.99/bw@8.0/noparent@true
addZoomFreq = freq@8352.99/bw@8.0/noparent@true
addZoomFreq = freq@8512.99/bw@8.0/noparent@true
addZoomFreq = freq@8732.99/bw@8.0/noparent@true
addZoomFreq = freq@8852.99/bw@8.0/noparent@true
addZoomFreq = freq@8912.99/bw@8.0/noparent@true
addZoomFreq = freq@8932.99/bw@8.0/noparent@true
}
```

Running report.py, I get an included v2d file with the corresponding section
```
ZOOM zoom
{ 
addZoomFreq = noparent@true
addZoomFreq = noparent@true
addZoomFreq = noparent@true
addZoomFreq = noparent@true
addZoomFreq = noparent@true
addZoomFreq = noparent@true
addZoomFreq = noparent@true
addZoomFreq = noparent@true
}
```

This omits important info from the zoom section, so I consider this a bug.

I also have datastream format definitions on the form
`format = VDIF/8032/2`
which is presented by report.py simply as
`  format = 2`

So, it seems that in v2d file strings with / included, only the text after the last / is copied to by report.py.

**Comment by phillip-haftings-usno** [comment]
This seems to be a side-effect of the way the report tries to strip out (obscure) local file paths.
I need to make it a little smarter so that zoom band definitions are properly distinguished from file paths.

**Comment by varenius** [comment]
Sounds good. Please note that the most serious case, however, is likely the “format=” strings being treated wrongly if they contain slashes. Zoom bands are more “special case”, which we use a lot for our mixed-mode local-tie experiments. Also good to fix, but less common at correlators I guess.

**Comment by phillip-haftings-usno** [comment]
This should now be fixed in the `main` branch as of [this revision](https://github.com/USNO-VLBI/main/commit/dfbb54839a25feb43d2c41d3c9469746cc461c1c).

**Comment by varenius** [comment]
Great - I can confirm it works. Could we perhaps get a new release with this in?

**Comment by phillip-haftings-usno** [comment]
Sure, I included in release v1.1.3 just now.
